docs/configure/nodes.md

```diff
@@ -31,20 +31,36 @@ scripts from inside that directory.
 
 ## Netdata's configuration files
 
-Upon installation, the Netdata config directory contains a few files and directories.
-
--   `netdata.conf` is the main configuration file. This is where you'll find most configuration options. This doc won't
-    go into exhaustive detail about each setting. You can read descriptions for each in the [daemon config
-    doc](/daemon/config/README.md).
--   `orig` is a symbolic link to the directory `/usr/lib/netdata/conf.d`, which contains stock configuration files.
-    Stock versions are copied into the config directory when opened with `edit-config`. _Do not edit the files in
-    `/usr/lib/netdata/conf.d`, as they are overwritten by updates to the Netdata Agent._
--   `edit-config` is a shell script used for [editing configuration files](#use-edit-config-to-edit-netdataconf).
--   `go.d/`, `python.d/`, `charts.d/`, `node.d`/, and `custom-plugins.d/`, which are directories for each of Netdata's
-    [orchestrators](/collectors/plugins.d/README.md). These directories can each contain additional `.conf` files for
-    configuring specific collectors.
-
-## Use `edit-config` to edit `netdata.conf`
+Upon installation, the Netdata config directory contains a few files and directories. It's okay if you don't see all
+these files in your own Netdata config directory, as the next section describes how to edit any that might not already
+exist.
+
+- `netdata.conf` is the main configuration file. This is where you'll find most configuration options. Read descriptions
+  for each in the [daemon config doc](/daemon/config/README.md).
+- `edit-config` is a shell script used for [editing configuration files](#use-edit-config-to-edit-configuration-files).
```

Comment:
You cited the `.conf` extension here, probably you only need to add a link above.

Reply:
Where would you like me to link?

Reply:
It is not necessary, the changes you made were good. Thanks!